front/src/applications/stdcmV2/components/StdcmConsist.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useState } from 'react';
+
+// import { Input } from '@osrd-project/ui-core';
+
+import { type LightRollingStockWithLiveries } from 'common/api/osrdEditoastApi';
+import { type SelectOptionObject } from 'common/BootstrapSNCF/SelectSNCF';
+import { useOsrdConfActions } from 'common/osrdContext';
+import SpeedLimitByTagSelector from 'common/SpeedLimitByTagSelector/SpeedLimitByTagSelector';
+import { useStoreDataForSpeedLimitByTagSelector } from 'common/SpeedLimitByTagSelector/useStoreDataForSpeedLimitByTagSelector';
+import RollingStock2Img from 'modules/rollingStock/components/RollingStock2Img';
+import { useStoreDataForRollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector/useStoreDataForRollingStockSelector';
+import useFilterRollingStock from 'modules/rollingStock/hooks/useFilterRollingStock';
+import type { StdcmConfSliceActions } from 'reducers/osrdconf/stdcmConf';
+import { useAppDispatch } from 'store';
+
+import StdcmCard from './StdcmCard';
+import StdcmSuggestions from './StdcmSuggestions';
+
+interface StdcmSuggestionsConsistOption
+  extends SelectOptionObject,
+    Omit<LightRollingStockWithLiveries, 'id'> {
+  value: LightRollingStockWithLiveries;
+}
+
+const ConsistCardTitle = ({
+  rollingStock,
+}: {
+  rollingStock?: LightRollingStockWithLiveries | null;
+}) => {
+  if (!rollingStock) return null;
+
+  return (
+    <div className="stdcm-v2-consist-img w-75 d-flex justify-content-end">
+      <RollingStock2Img rollingStock={rollingStock} />
+    </div>
+  );
+};
+
+const StdcmConsist = ({ isPending = false }: { isPending?: boolean }) => {
+  const { speedLimitByTag, speedLimitsByTags, dispatchUpdateSpeedLimitByTag } =
+    useStoreDataForSpeedLimitByTagSelector();
+
+  const { updateRollingStockID } = useOsrdConfActions() as StdcmConfSliceActions;
+  const dispatch = useAppDispatch();
+
+  const { rollingStock } = useStoreDataForRollingStockSelector();
+
+  const [isSelectChanged, setIsSelectChanged] = useState(false);
```

Comment:
I think the name of the variable is not very clear, what do you think of rsSelectedByClick ? or rsJustSelectedByClick ?
```suggestion
  const [rsSelectedByClick, setRsSelectedByClick] = useState(false);
```